javaharness/java/arcs/android/AndroidArcsEnvironment.java

```diff
@@ -146,6 +146,7 @@ public WebResourceResponse shouldInterceptRequest(WebResourceRequest request) {
     });
 
     webView.addJavascriptInterface(this, "DeviceClient");
+    webView.addJavascriptInterface(new ArcsSystemTrace(), "systemTraceApis");
```

Comment:
Hmm, it makes sense to maintain capitalized naming, I'll change it.
As for why not just SystemTrace, I thought about it. To differentiate it from the @SystemTrace decorator in JS, SystemTraceApis can tell it's apis imported from somewhere like Java world or other delegations instead of built-in functions in JS.